ocamlformat-help.txt

```diff
@@ -579,6 +579,9 @@ OPTIONS
            making some syntactic gotchas visually obvious. The janestreet
            profile is used at Jane Street. The default value is default.
 
+       --partial
+           Ignore invalid (unparsable) parts of the input.
```

Comment:
I think it'll be worth expanding a bit, explaining that formattable bits get formatted, and parts with errors are passed verbatim. Also this can mentioned that it is a pretty preliminary implementation.

Reply:
I improved the doc a bit